bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api1/Shelly1ApiJsonDTO.java

```diff
@@ -577,6 +578,7 @@ public static class ShellySettingsGlobal {
         public Boolean wifiRecoveryReboot; // FW 1.10+
         @SerializedName("ap_roaming")
         public ShellyApRoaming apRoaming; // FW 1.10+
+        public Boolean rangeExtender; // Gen2: Range extender
```

Comment:
this should be `boolean`

Reply:
No, those are the mappings from JSON returned from the device into internal structures. The value might be null if the device doesn't support that feature / doesn't return the value